rust/private/rust.bzl

```diff
@@ -938,3 +938,6 @@ rust_benchmark(
 Run the benchmark test using: `bazel run //fibonacci:fibonacci_bench`.
 """,
 )
+
+def name_to_crate_name(name):
```

Comment:
Can you add a docstring here and give some context as to why it exists?

Reply:
Done :)